commands.adoc

```diff
@@ -223,10 +223,18 @@ Similarly, you can use `$ vagrant service-manager status openshift`,
 
 If you need to install the client binary for a specified service, use: +
 
-`vagrant service-manager install-cli [service]` +
+`vagrant service-manager install-cli [service] [options] [arg]` +
```

Comment:
Do you really need '[arg]' here? The value for an option is really encapsulated in [options] itself, right? I guess one can do something like '--cli-version=' to indicate that the option takes a value.